### DISCUSSION TITLE
List for Sequels like MCU

### DISCUSSION THREAD

**Discussion by kooolarpan** [trakt/showly#389] [Ideas / Feature Requests]
I have used Letterboxd which is an app where we can find the reviews, ratings and lists of sequels.

I really liked the lists feature where we can get the list of a sequels for example MCU, X-MEN, etc.

![Screenshot_2021-01-19-19-15-32-208_com.letterboxd.letterboxd.jpg](https://user-images.githubusercontent.com/33162706/105043090-26a26e80-5a8d-11eb-91b9-ffb76742f249.jpg)

**Comment by michaldrabik** [comment]
I think this is like a user's custom list: https://trakt.tv/users/c3li4/lists/mcu?sort=released,desc

I do have plan to add custom user shows and movies lists support later this year : )

**Reply by kooolarpan** [reply]
Great job. Appreciated :)

**Comment by blackholeearth** [comment]
This has been implemented.

Duplicate #627
